code/game/machinery/turrets.dm

```diff
@@ -520,6 +520,13 @@
 		return
 	return ..()
 
+/obj/machinery/turretid/CtrlClick(mob/user) //Lock the device
```

Comment:
this actually has no sanity whatsoever, so you can do this while dead or unconscious

no, really (same for the other)